proto/vmexec.proto

```diff
@@ -62,6 +67,49 @@ message ExecResponse {
   google.rpc.Status status = 4;
 }
 
+message ExecStreamedRequest {
+  oneof Message {
+    // Begins command execution. If there is no stdin to be sent, then clients
+    // should close the send-side of the stream immediately after sending this
+    // message.
+    ExecRequest start = 1;
+
+    // Bytes to be written to stdin of the executed process. In order to use
+    // this field, `open_stdin` must be set in the initial start request,
+    // otherwise execution will be canceled and an INTERNAL error will be
+    // returned. Clients should close the send-side of the stream once there is
+    // no more input to be sent (this will close the stdin pipe).
+    bytes stdin = 2;
+  }
+}
+
+message ExecStreamedResponse {
+  oneof Message {
```

Comment:
@tylerwilliams I started thinking about the approach where we call Start() and it returns an ID that you can use to attach stdin, get stdout/stderr, and get Stats(), but I realized it might be more complex than this single streaming approach because it would involve 6 separate RPCs, like Start(), ReadStdout(), ReadStderr(), WriteStdin(), PollStats(), Wait(), and it seemed tricky to get all those separate RPCs to talk to each other and coordinate state, and each one would have a different ctx etc. This single RPC approach seemed simpler but maybe there's other factors I'm not considering